fenix/app/src/main/java/org/mozilla/fenix/onboarding/OnboardingFragment.kt

```diff
@@ -249,12 +250,14 @@ class OnboardingFragment : Fragment() {
                 },
             ),
         )
-        return FxNimbus.features.junoOnboarding.value().cards.values.toPageUiData(
-            privacyCaption,
-            showDefaultBrowserPage,
-            showNotificationPage,
-            showAddWidgetPage,
-            jexlConditions,
-        ) { condition -> jexlHelper.evalJexlSafe(condition) }
+        return jexlHelper.use {
+            FxNimbus.features.junoOnboarding.value().cards.values.toPageUiData(
+                privacyCaption,
+                showDefaultBrowserPage,
+                showNotificationPage,
+                showAddWidgetPage,
+                jexlConditions,
+            ) { condition -> jexlHelper.evalJexlSafe(condition) }
```

Comment:
Could we do this 👇  instead?

```
{ condition -> 
  jexlHelper.use {
    evalJexlSafe(condition) }
  }
``` 
So we don't have to wrap the whole mapper call inside the use 🤔

Reply:
No; the jexl helper designed to be used for a collection of JEXL expressions, rather than one then done.

There are two FFI calls for each creation, and one for the destruction, then an internal cache to to memoize the `evalJexl` call.

Reply:
Gotcha! I had forgotten that `toPageUiData` is mapping a list to another list and is calling this function for each element, just double checked, this looks good, thanks for the clarification! 👍